components/dashboard/src/projects/Prebuild.tsx

```diff
@@ -181,7 +181,9 @@ export default function () {
                     ) : prebuild?.status === "available" ? (
                         <a
                             className="my-auto"
-                            href={gitpodHostUrl.withContext(`${prebuild?.info.changeUrl}`).toString()}
+                            href={gitpodHostUrl
+                                .withContext(`open-prebuild/${prebuild?.info.id}/${prebuild?.info.changeUrl}`)
+                                .toString()}
                         >
                             <button>New Workspace ({prebuild?.info.branch})</button>
```

Comment:
I guess the `(branch)` part of the button label no longer makes sense when we're opening the prebuild.

Maybe let's remove it?

```suggestion
                            <button>New Workspace</button>
```

Or clarify it?

```suggestion
                            <button>New Workspace With This Prebuild</button>
```